backend/src/langchain/langchain.module.ts

```diff
@@ -1,10 +1,23 @@
 import { Module } from "@nestjs/common";
 import { ChatOpenAI } from "@langchain/openai";
+import { ChatOllama } from "@langchain/ollama";
 import { BaseChatModel } from "@langchain/core/language_models/chat_models";
 
 const chatModelFactory = {
 	provide: "ChatModel",
-	useFactory: () => new ChatOpenAI({ modelName: "gpt-4o-mini" }) as BaseChatModel,
+	useFactory: () => {
+		const modelType = process.env.YORKIE_INTELLIGENCE;
+		if (modelType === "gemma2:2b"){
+			return new ChatOllama({
+				model: modelType,
+				checkOrPullModel: true,
+				streaming: true,
+			});
```

Comment:
How about add `baseUrl` parameter and set the value from `.env` file?
Then, users can change the ollama environment.